bam-reader/src/main/java/org/victorchang/SAMRecordGenerator.java

```diff
@@ -0,0 +1,26 @@
+package org.victorchang;
+
+import htsjdk.samtools.SAMRecord;
+
+import java.io.DataInput;
+import java.io.IOException;
+import java.util.function.Consumer;
+
+public class SAMRecordGenerator implements BamRecordHandler {
```

Comment:
Replace `SAM` with `Sam`? To make it consistent with `BamRecordHandler` etc. Same with other `SAM*` classes.